src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodIgnoreThrows.java

```diff
@@ -0,0 +1,71 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.function.Function;
+
+public class InputJavadocMethodIgnoreThrows {
+
+   /**
+     * Ignore try block, but keep catch and finally blocks.
+     *
+     * @param s String to parse
+     * @return A positive integer
+     */
+    private static int parsePositiveInt(String s) {
+        try {
+            int value = Integer.parseInt(s);
+            if (value <= 0) {
+                throw new NumberFormatException(value + " is negative/zero"); // ok, try
+            }
+            return value;
+        } catch (NumberFormatException ex) {
+            throw new IllegalArgumentException("Invalid number", ex); // violation, catch
+        } finally {
+            throw new IllegalStateException("Should never reach here"); // violation, finally
+        }
+    }
+
+    /**
+     * Ignore everything inside lambda.
+     *
+     * @param maxLength Max length
+     * @return A function to truncate string
+     */
+    private static Function<String, String> getTruncateFunction(int maxLength) {
+        return s -> {
+            if (s == null) {
+                throw new IllegalArgumentException("Cannot truncate null"); // ok, inside lambda
+            }
+            return s.length() > maxLength ? s.substring(0, maxLength) : s;
+        };
+    }
+
+    /**
+     * Try-with-resources should also be ignored if there is a catch block.
+     *
+     * @param input file to read
+     */
+    private static void ignoreTryWithResources(String input) {
+        try (BufferedReader in = new BufferedReader(new FileReader(input))) {
+            System.out.println(in.readLine());
+        } catch (IOException e) {
```

Comment:
you should throw exception from `try` to test the check

Reply:
Added 2 types of `throws`, an `IOException` which is to be caught and an `IllegalArgumentException` which is a false negative example.